canonical-web-and-design/ubuntu.com
# Advantage shop checkout markup

## Done

- Adapted the renewal modal to accommodate multiple products and display information about each during checkout.

**Note:** some of this code is purely for demo purposes and will be removed when we have real data coming in from the contracts API.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/advantage/subscribe
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Select your products and their quantities
- Click "Buy"
- See that the modal matches the [brief](https://docs.google.com/document/d/1K65n3NpZCh7EWDXKLhG31seGJ2jKbW6lqw3BwV2mlnU/edit#heading=h.mz73r7dju49w)
- Add payment method details and click "Continue", see that the second page looks correct
- Click "Cancel" (the "Pay" button does not work for this demo)
- Select a different combination of products, click "Buy", see that the summary has been updated.


## Issue / Card

Fixes #7573
